Terminal Commands.md

```diff
@@ -22,8 +28,18 @@ Some commands can be found in the `bin` directory.
 `mkdir` - makes a directory<br/>
 `rmdir` - removes an **empty** directory<br/>
 
+`cp file1 file2` - used to copy file1 to file2.<br>
+`mv file1 file2` - used to move file1 to file2.
+
+- **Trivia:** `mv oldfilename newfilename` is the best method to rename a file.
+
 `grep <text to search> <files to search in>`
 
+`cut` is used to cut a part of a file<br>
```

Comment:
Your repo, your rules. You could accept and make subtle changes on your own too. Would save everbody's time -.-